specification/eventgrid/resource-manager/Microsoft.EventGrid/preview/2019-02-01-preview/EventGrid.json

```diff
@@ -2274,7 +2638,9 @@
                     "description": "The event delivery schema for the event subscription.",
                     "enum": [
                         "EventGridSchema",
+                        "InputEventSchema",
```

Comment:
Please remove this value (as this is deprecated and has been replaced by CustomInputSchema.)

Reply:
removed